test/webrick/test_httprequest.rb

```diff
@@ -118,6 +118,27 @@ def test_bare_lf_header
     }
   end
 
+  def test_header_vt_ff_whitespace
+    msg = <<-_end_of_message_
+      GET / HTTP/1.1\r
+      Foo: \x0b1\x0c\r
+      \r
+    _end_of_message_
+    req = WEBrick::HTTPRequest.new(WEBrick::Config::HTTP)
+    req.parse(StringIO.new(msg.gsub(/^ {6}/, "")))
```

Comment:
Do you mind updating this to use squiggly heredocs?

Reply:
Updated and rebased against master.